src/QsCompiler/LlvmBindings/DebugInfo/DILocalVariable.cs

```diff
@@ -0,0 +1,31 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="DILocalVariable.cs" company="Ubiquity.NET Contributors">
+// Copyright (c) Ubiquity.NET Contributors. All rights reserved.
+// </copyright>
+// -----------------------------------------------------------------------
+
+using LLVMSharp.Interop;
+
+namespace Ubiquity.NET.Llvm.DebugInfo
+{
+    /// <summary>Debug information for a local variable</summary>
+    /// <seealso href="xref:llvm_langref#dilocalvariable">LLVM DILocalVariable</seealso>
+    public class DILocalVariable
+        : DIVariable
+    {
+        // ReSharper disable IdentifierTypo
+        /* TODO: non-operand properties
```

Comment:
This is a bit of a nitpick, since these were present in the original Ubiquity code, but would you mind removing these TODO lists from the DI classes? In our case, there isn't really an intent to come back and fill these in, so they aren't true to-do items for this code base.

Reply:
https://github.com/microsoft/qsharp-compiler/pull/1026/commits/8505bc61cebaf92e8beca7fd2433edb56f459834